src/transformers/models/time_series_transformer/configuration_time_series_transformer.py

```diff
@@ -52,7 +52,7 @@ class TimeSeriesTransformerConfig(PretrainedConfig):
             The size of the target variable which by default is 1 for univariate targets.
         scaling (`bool`, *optional* defaults to `True`):
             Whether to scale the input targets.
-        lags_seq (`list` of `int`  *optional* defaults to `[1, 2, 3, 4, 5, 6, 7]`):
+        lags_seq (`list[int]`, *optional*;, defaults to `[1, 2, 3, 4, 5, 6, 7]`):
```

Comment:
```suggestion
        lags_seq (`list[int]`, *optional*, defaults to `[1, 2, 3, 4, 5, 6, 7]`):
```
Sorry made a typo here